### ISSUE TITLE
Firefox Unregister qz:launch on Uninstall

### ISSUE THREAD

**Issue by tresf** [qzind/tray#265]
For Firefox, we register a URI handler for `qz:` protocol, but this gets left behind after uninstall.

https://github.com/qzind/tray/blob/2.0/ant/firefox/firefox-config.cfg.in#L60-L80

**Comment by tresf** [comment]
I tried the obvious `hservice.remove(...);` to no avail.  It doesn't error, but it also doesn't remove it.  I have no idea how to do this... 

```js
        // De-registers the specified protocol
        function unregisterProtocol() {
            var trayApp = "";
            if (!trayApp) { return; }
            try {
                var hservice = Components.classes["@mozilla.org/uriloader/handler-service;1"].getService(Components.interfaces.nsIHandlerService);
                var pservice = Components.classes["@mozilla.org/uriloader/external-protocol-service;1"].getService(Components.interfaces.nsIExternalProtocolService);
                hservice.remove(pservice.getProtocolHandlerInfo("qz"));
            } catch(ignore) {}
        }
```

**Comment by tresf** [comment]
Turns out I was over-thinking this... Firefox on Windows and Linux doesn't register the protocol manually, but MacOS does.  Windows and Linux use the protocol information that was registered with the system through the [registry](https://github.com/qzind/tray/blob/7e69350b04a1d5327910b186f403a9428db5656d/ant/windows/windows-packager.nsi.in#L80) or [mimetype registration](https://github.com/qzind/tray/blob/7e69350b04a1d5327910b186f403a9428db5656d/ant/linux/linux-installer.sh.in#L125).  For Windows, this was fixed inadvertently in #266.  Linux is still broken but should be as simple as adding `update-desktop-database` to the Linux uninstaller script (we will track that separately if needed).

Confirmed working on MacOS.

![screen shot 2018-02-20 at 11 08 24 pm](https://user-images.githubusercontent.com/6345473/36462717-059d7f4a-1693-11e8-857e-0176937a4298.png)
